src/In.ProjectEKA.HipService/Link/Model/Auth.cs

```diff
@@ -1,16 +1,18 @@
 namespace In.ProjectEKA.HipService.Link.Model
 {
-    public class AuthInit
+    public class Auth
     {
-        public AuthInit(string transactionId, AuthType authType, Meta meta)
+        public Auth(string transactionId, Meta meta,Mode mode)
         {
             TransactionId = transactionId;
-            AuthType = authType;
+         //   AuthType = authType;
             Meta = meta;
+            Mode = mode;
         }
 
         public string TransactionId { get; }
-        public AuthType AuthType { get; }
+       // public AuthType AuthType { get; }
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6173534)